### ISSUE TITLE
Logstash 2.3.2 not working with Elasticsearch Output

### ISSUE THREAD

**Issue by suyograo** [elastic/logstash#5188]
Latest 2.3.2 here: https://download.elastic.co/logstash/logstash/logstash-2.3.2.tar.gz

```
ubuntu@ip-10-0-1-10:/tmp/logstash-2.3.2$ cp ~/logstash-2.3.2/complete.conf .
ubuntu@ip-10-0-1-10:/tmp/logstash-2.3.2$ bin/logstash -f complete.conf
fetched an invalid config {:config=>"input {\n  stdin { }\n}\n\nfilter {\n  grok {\n    match => {\n      \"message\" => '%{IPORHOST:clientip} %{USER:ident} %{USER:auth} \\[%{HTTPDATE:timestamp}\\] \"%{WORD:verb} %{DATA:request} HTTP/%{NUMBER:httpversion}\" %{NUMBER:response:int} (?:-|%{NUMBER:bytes:int}) %{QS:referrer} %{QS:agent}'\n    }\n  }\n\n  date {\n    match => [ \"timestamp\", \"dd/MMM/YYYY:HH:mm:ss Z\" ]\n    locale => en\n  }\n\n  geoip {\n    source => \"clientip\"\n  }\n\n  useragent {\n    source => \"agent\"\n    target => \"useragent\"\n  }\n}\n\noutput {\n  stdout { codec => rubydebug }\n  elasticsearch {\n    hosts => \"localhost\"\n  }\n}\n\n\n", :reason=>"\n\n\tyou might need to reinstall the gem which depends on the missing jar or in case there is Jars.lock then resolve the jars with `lock_jars` command\n\nno such file to load -- org/apache/httpcomponents/httpcore/4.4.1/httpcore-4.4.1 (LoadError)", :level=>:error}
```

Initially I thought I was hitting https://github.com/elastic/logstash/issues/4818 but I do see the `.mvn` and the xml file here:

```
ubuntu@ip-10-0-1-10:/tmp/logstash-2.3.2$ ls -al ./vendor/bundle/jruby/1.9/gems/ruby-maven-3.3.10/.mvn/extensions.xml
-rw-r--r-- 1 ubuntu ubuntu 383 Apr 25 17:35 ./vendor/bundle/jruby/1.9/gems/ruby-maven-3.3.10/.mvn/extensions.xml
```

Config:

```
input {
  stdin { }
}

output {
  stdout { codec => rubydebug }
  elasticsearch {
    hosts => "localhost"
  }
}
```

**Comment by ph** [comment]
Small brain dump here.

From the jenkins build job

```
  jar dependencies for manticore-0.5.5-java.gemspec . . .
Installing gem 'ruby-maven' . . .

RuntimeError: there was an error installing 'ruby-maven (~> 3.3.3)' . please install it manually: #<Gem::DocumentError: RDoc is not installed: no such file to load -- rdoc/rdoc>
Fetching gem metadata from https://rubygems.org/.......
Fetching version metadata from https://rubygems.org/...
Fetching dependency metadata from https://rubygems.org/..
Using addressable 2.3.8
```

Why we have an rdoc related error during the installing? By default bundler doesn't install rdoc.

**Comment by suyograo** [comment]
Fixed in #5189 by downgrading jruby-kafka to 1.5.0
